part of the fix for RC bug #634577 caused by libtool:
	as the build is sanitized by using dh_auto_build, 
	`debian/pam_ssh.version' must be passed to LD via LDFLAGS set up in Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -25,7 +25,7 @@
 # $Id: Makefile.am,v 1.18 2004/02/20 14:57:23 akorty Exp $
 
 lib_LTLIBRARIES			= pam_ssh.la
-pam_ssh_la_LDFLAGS		= -avoid-version -module
+pam_ssh_la_LDFLAGS		= -avoid-version -module -Wl,--version-script,debian/pam_ssh.version
 pam_ssh_la_LIBADD		= @LTLIBOBJS@
 pam_ssh_la_SOURCES		= atomicio.c atomicio.h authfd.c authfd.h   \
 				  authfile.c authfile.h bufaux.c bufaux.h   \
